packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -17,7 +18,7 @@ import {
 } from 'roosterjs-editor-types';
 import type { CompatibleTableOperation } from 'roosterjs-editor-types/lib/compatibleTypes';
 
-const DEFAULT_FORMAT: Required<TableFormat> = {
+const DEFAULT_FORMAT: Partial<TableFormat> = {

```

Comment:
Sorry I misunderstood.

So here we should keep it as Required, and you may need to let the verticalAlign allow an empty value. e.g.
```ts
interface TableFormat {
 ...
 verticalAlign?: 'top' | 'bottom' | 'middle' | '';
}
```

Then put the empty string value here, which means it is not specified. Then when apply the value, check if the cell as vertical align override and if the format value is not empty, e.g.
```ts
if (!override[x][y] && format.verticalAlign != '' ) {
  ...
}
```

Reply:
or use null as the empty value, should also ok

Reply:
The implementation of alignment in non-content model is done differently, by adding it to the row on creation on insertTable.